Mathlib/Data/Fintype/Basic.lean

```diff
@@ -940,10 +940,12 @@ instance PLift.fintypeProp (p : Prop) [Decidable p] : Fintype (PLift p) :=
 instance Prop.fintype : Fintype Prop :=
   ⟨⟨{True, False}, by simp [true_ne_false]⟩, by simpa using em⟩
 
-@[simp]
```

Comment:
Why did you remove this `@[simp]`?
I am not saying you shouldn't remove it, but I'd like to have more details about the reasons.

Reply:
This was causing problems with simp confluence, where lemmas on `Set.univ` wouldn't fire when written in this form. My proof breaks with the simp attribute active.